src/scss/components/choice-tile/_index.scss

```diff
@@ -0,0 +1,218 @@
+  }
+
+  &.hxSm {
+    $pad: 0.75rem;
+
+    padding: $pad;
+
+    > p {
+      display: none;
+    }
+
+    hx-icon[type="checkmark"] {
+      right: $pad;
+      top: $pad;
+    }
+  }
+
+  &.hxLg {
+    $pad: 1.75rem;
+
+    padding: $pad;
+
+    hx-icon[type="checkmark"] {
+      right: $pad;
+      top: $pad;
+    }
+  }
+
+  > header + *,
+  .hx-tile-icon + * {
+    margin-top: 0.5rem;
+  }
+
+  .hx-tile-icon {
+    hx-icon {
+      font-size: 1.75rem;
+    }
+  }
+
+  hx-icon[type="checkmark"] {
+    display: none;
+    font-size: 1rem;
+    right: 1.25rem;
+    top: 1.25rem;
+  }
+
+  &.hxSubdued {
+    @include hxChoiceTile--subdued;
+
+    &:hover {
+      @include hxChoiceTile--subdued($pseudo-state: hover);
+    }
+  }
+}
+
+.hxRow > label.hxChoice {
+  @include grid-column;
+}
+
+// Layout & Geometry
+// ========== COMMON ==========
+label.hxChoice {
+  position: relative;
+}
+
+label.hxChoice > input {
+  @include hxInput(visuallyHidden);
+
+  &:focus {
+    @include reset-focus;
+  }
+
+  &:checked + hx-tile {
+    hx-icon[type="checkmark"] {
+      display: block;
+      position: absolute;
+    }
+  }
+}
+
+// ========== SUBDUED ==========
+label.hxChoice > input + hx-tile:hover.hxSubdued {
+  @include hxChoiceTile--subdued($pseudo-state: hover);
+}
+
+label.hxChoice > input[disabled] + hx-tile.hxSubdued,
+label.hxChoice > input[invalid] + hx-tile.hxSubdued {
+  opacity: 1;
+}
+
+// ========== DEFAULT ==========
+// NOTE: These styles may be able to go into default hx-tile defs
+label.hxChoice > input + hx-tile {
+  background-color: $gray-0;
+  border: solid 1px $gray-400;
+  color: $gray-900;
+  cursor: pointer;
+
+  .hx-tile-icon {
+    color: $gray-700;
+  }
+}
+
+label.hxChoice > input + hx-tile:hover {
+  @include hxChoiceTile($pseudo-state: hover);
+}
+
+// same as hover, with focus glow
+label.hxChoice > input:focus + hx-tile {
+  @include hxChoiceTile($pseudo-state: focus);
+}
+
+// ========== CHECKED ==========
+label.hxChoice > input:checked + hx-tile {
+  border-color: $cyan-900;
+  color: $cyan-900;
+
+  .hx-tile-icon {
+    color: $cyan-900;
+  }
+
+  > p {
+    color: $gray-900;
+  }
+
+  // DEPRECATED: remove in v1.0.0
+  hx-tile-description {
+    color: $gray-900;
+  }
+}
+
+// ========== INVALID ==========
+label.hxChoice > input[invalid] + hx-tile {
+  border-color: $red-900;
+  border-width: 2px;
+  color: $red-900;
+
+  > p {
+    color: $gray-900;
+  }
+
+  // DEPRECATED: remove in v1.0.0
+  hx-tile-description {
+    color: $gray-900;
+  }
+
+  .hx-tile-icon {
+    color: inherit;
+  }
+}
+
+label.hxChoice > input[invalid] + hx-tile:hover {
+  @include hxChoiceTile--invalid($pseudo-state: hover);
+}
+
+label.hxChoice > input[invalid]:focus + hx-tile {
+  @include hxChoiceTile--invalid($pseudo-state: focus);
+}
+
+// ========== DISABLED ==========
+label.hxChoice > input[disabled] + hx-tile,
+label.hxChoice > input[disabled] + hx-tile:hover {
+  @include hxChoiceTile--disabled($pseudo-state: hover);
+}
```

Comment:
Missing styles originally present in LESS file... 

https://github.com/HelixDesignSystem/helix-ui/blob/master/src/less/components/choice-tile/index.less#L207-L218

* missing styles should be defined in this file, not the mixin